testng-core/src/main/java/org/testng/TestNG.java

```diff
@@ -353,7 +353,8 @@ private OverrideProcessor getProcessor() {
       TestNamesMatcher testNamesMatcher = new TestNamesMatcher(s, m_testNames);
       List<String> missMatchedTestname = testNamesMatcher.getMissMatchedTestNames();
       if (!missMatchedTestname.isEmpty()) {
-        throw new TestNGException("The test(s) <" + missMatchedTestname + "> cannot be found.");
+        throw new TestNGException(
```

Comment:
It could be nice to integrate the check directly into `getSuitesMatchingTestNames`.

Reply:
done